clowder-framework/clowder2
# 132 reactive search routes when token expire logout

To test:
Set the token expiration in keycloak admin console (http://localhost:8080/keycloak) to 1 minute; and type in the search box see what happened after 1 minute
<img width="549" alt="image" src="https://user-images.githubusercontent.com/13950475/197645848-830d254e-92df-4c30-8cb3-b3813f12623f.png">
